phoenicis-containers/src/main/java/org/phoenicis/containers/GenericContainersManager.java

```diff
@@ -120,9 +120,7 @@ public void fetchContainers(Consumer<List<ContainerCategoryDTO>> onSuccess, Cons
     @Override
     public void deleteContainer(ContainerDTO container, Consumer<ContainerDTO> onSuccess, Consumer<Exception> onError) {
         try {
-            final File containerPath = new File(container.getPath());
-
-            this.fileUtilities.remove(containerPath);
+            this.fileUtilities.remove(container.getPath());
```

Comment:
I'm not sure if we should call the functions which we export for ease of use to the JS scripts internally in our main Application. I think it would be cleaner to directly call Java APIs to perform the deletion, or we forward the call somehow to a JS script.

Reply:
I agree.

Reply:
Done